Runtime/InputSystemActionPrompts/InputDevicePromptSystem.cs

```diff
@@ -128,6 +128,13 @@ public static string InsertPromptSprites(string inputText)
             foreach (var tag in foundTags)
             {
                 var replacementTagText = GetActionPathBindingTextSpriteTags(tag);
+                
+                //if PromptSpriteFormatter is empty for some reason return the text as if formatter was {SPRITE} (normally)
+                var promptSpriteFormatter = s_Settings.PromptSpriteFormatter == "" ? "{SPRITE}" : s_Settings.PromptSpriteFormatter;
```

Comment:
Actually,  sorry a diff suggestion: 
1. Move "{SPRITE}" to be a public const string in InputDevicePromptSystem and use in all places
2. Check that the s_Settings.PromptSpriteFormatter contains {SPRITE} on Initialise. If it doesn't, log an error to the effect that it's missing.

Reply:
Hey man you're totally right on the suggestions! Sorry been a bit busy but will try to get to this asap so we can get this merged in asap!